packages/web-app-files/src/store/mutations.js

```diff
@@ -6,9 +6,6 @@ import { renameResource } from '../helpers/resources'
 import { ShareTypes } from '../helpers/share'
 
 export default {
-  LOAD_SPACES(state, spaces) {
-    state.spaces = spaces
-  },
   /**
    * Updates a single space field. If the space with given id doesn't exist nothing will happen.
    *
```

Comment:
Like @JanAckermann already posted, the following mutations `UPDATE_SPACE_FIELD`, `UPSERT_SPACE`, `REMOVE_SPACE`, `CLEAR_SPACES` should move over to the new store module as well.

Reply:
✅